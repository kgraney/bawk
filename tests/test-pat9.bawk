/* LE defaults to false (0) */

/ 091938af / {
	print(100);
	print(RP);
}

LE = 1;

/ af381909 / {
	print(200);
	print(RP);
}

/ 09 19 38 af / {
	print(300);
	print(RP);
}

/ af 38 19 09 / {
	print(401);
	print(RP);
}

LE = 0;

/ 091938af / {
	print(500);
	print(RP);
}


/ 09 19 38 af / {
	print(600);
	print(RP);
}