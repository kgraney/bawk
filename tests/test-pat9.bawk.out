100
72
200
72
300
72
500
72
600
72
